### PR TITLE
Fix python 2-3 cross-compatibility issue

### PR DESCRIPTION
**Pull request by kbairak**

### CODE REVIEW

File: ucf.py [hunk 1]
```diff
@@ -145,7 +145,9 @@ def save(self, filename=None):
         
         if self.rootfiles:
             container = _build_container(self.rootfiles)
-            self.meta[CONTAINER] = container.encode(UTF8)
+            if isinstance(container, unicode):
```
Comment (davidwtbuxton) [inline ucf.py]: `unicode` isn't a built-in in Python 3.x.

**Comment by davidwtbuxton** [comment]
Hi @kbairak 

Thanks for the PR! Hope this library is of use.

Are you using this with Python 2?

Dave

File: ucf.py (hunk 1 above)
Comment (davidwtbuxton) [reply ucf.py]: Oh wow, I see I put a `unicode = str` compatibility thing in there, so that this will work fine with Py3. My bad.

**Comment by kbairak** [comment]
We used to use it with python2. Now we want to be python 2-3 cross-compatible.

To be honest, I could have looked more thoroughly for other similar issues, but I only fixed the one that caused our test build to fail.

**Comment by davidwtbuxton** [comment]
I am happy to accept more PRs for Py3 compatibility as you find them.

Do you prefer to install via PyPI, in which case I should create proper releases?

What versions of Python 3 are you targeting?

(I am still coming to terms with the idea that this package I created ages back for a project to handle Adobe InDesign XML files, which ended up never using this code, which I haven't looked at since then, was actually used by someone.)

**Comment by kbairak** [comment]
> Do you prefer to install via PyPI, in which case I should create proper releases?

It wouldn't hurt. For now I am happy to link my setup.py to your github repo instead of keeping my own but I would prefer to pull it from PyPI. Whatever you want.

Thank you for taking care of this.

**Pull request merged**